CHANGELOG.md

```diff
@@ -6,6 +6,67 @@ This file is generated automatically. Add individual changelog entries to the 'c
+  [#1721](https://github.com/tenzir/vast/pull/1721)
+
+### :gift: Features
+
+- The new option `vast.start.commands` allows for specifying an ordered list of VAST commands that run after successful startup. The effect is the same as first starting a node, and then using another VAST client to issue commands.  This is useful for commands that have side effects that cannot be expressed through the config file, e.g., starting a source inside the VAST server that listens on a socket or reads packets from a network interface.
+  [#1699](https://github.com/tenzir/vast/pull/1699)
+
+- The options `vast.plugins` and `vast.plugin-dirs` may now be specified on the command line as well as the configuration. Use the options `--plugins` and `--plugin-dirs` respectively.
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+- Add the reserved plugin name `bundled` to `vast.plugins` to enable load all bundled plugins, i.e., static or dynamic plugins built alongside VAST, or use `--plugins=bundled` on the command line. The reserved plugin name `all` causes all bundled and external plugins to be loaded, i.e., all shared libraries matching `libvast-plugin-*` from the configured `vast.plugin-dirs`.
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+- It's now possible to configure the VAST endpoint as an environment variable by setting `VAST_ENDPOINT`. This has higher precedence than setting `vast.endpoint` in configuration files, but lower precedence than passing `--endpoint=` on the command-line.
+  [#1714](https://github.com/tenzir/vast/pull/1714)
+
+- Plugins load their respective configuration from `<configdir>/vast/plugin/<plugin-name>.yaml` in addition to the regular configuration file at `<configdir>/vast/vast.yaml`. The new plugin-specific file does not require putting configuration under the key `plugins.<plugin-name>`. This allows for deploying plugins without needing to touch the `<configdir>/vast/vast.yaml` configuration file.
+  [#1724](https://github.com/tenzir/vast/pull/1724)
+
+### :beetle: Bug Fixes
+
+- VAST no longer crashes when querying for string fields with non-string values.
+  [#1685](https://github.com/tenzir/vast/pull/1685)
+
+- Building plugins against an installed VAST no longer requires manually specifying `-DBUILD_SHARED_LIBS=ON`. The option is now correctly enabled by default for external plugins.
+  [#1697](https://github.com/tenzir/vast/pull/1697)
+
+- The UDS metrics sink continues to send data when the receiving socket is recreated.
+  [#1702](https://github.com/tenzir/vast/pull/1702)
+
+- The `vast.log-rotation-threshold` option was silently ignored, causing VAST to always use the default log rotation threshold of 10 MiB. The option works as expected now.
+  [#1709](https://github.com/tenzir/vast/pull/1709)
+
+- Additional tags for the [tenzir/vast](https://hub.docker.com/r/tenzir/vast) Docker image for the release versions exist, e.g., `tenzir/vast:2021.05.27`.
```

Comment:
```suggestion
- Additional version tags for the [tenzir/vast](https://hub.docker.com/r/tenzir/vast) Docker image for the release versions exist, e.g., `tenzir/vast:2021.05.27`.
```

Reply:
Docker calls these tags, they are not versions. We simply use our version as a tag now, but that still does not make it a version tag.